Students/Semester2/lessons/students/1_basics/4_basic_math/4_basic_math.ipynb

```diff
@@ -107,9 +108,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```

Comment:
we have to figure out what this field means for users who pull in these changes, not sure if it means it would run this cell 3 times??

Reply:
This part is not part of the code on vs